test/compression.test

```diff
@@ -0,0 +1,133 @@
+#!/bin/sh
+
+# shellcheck source=testlib.sh
+. "${0%/*}/testlib.sh"
```

Comment:
This should be `. ./testlib.sh` to ensure out-of-tree builds work. (This weird hack was actually required for out-of-tree builds when there was no `testlib.sh.in`, but only `testlib.sh`.)